.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     steps:
       - run:
           name: Install ansible-lint
-          command: sudo pip install ansible-lint
+          command: sudo pip install ansible-lint==4.3.2
```

Comment:
quelle est la raison pour fixer la version?

Reply:
To prevent the build from suddenly failing because an update of `ansible-lint` added more error cases.